doc/Command_Index_internal.tex

```diff
@@ -678,35 +675,30 @@ \section{Gregorio\TeX{} Controls}
 Workhorse function behind \verb=\grechangestyle=.  Necessary because the internals of the definition are slightly different in \LaTeX\ and Plain\TeX.
 
 \macroname{\textbackslash gre@calculate@bolshift}{\#1\#2}{gregoriotex-spaces.tex}
-Macro used in \verb=\GreSyllable=. Sets \verb=\gre@skip@bolshift= to the left kern that should appear at the beginning of a line in case of a forced linebreak.  The goal of this left kern is to have all lines aligned on notes.
+Macro used in \verb=\GreSyllable=. Sets \verb=\gre@skip@bolshift= to the left kern that should appear at the beginning of a line in case of a forced linebreak.  The goal of this left kern is to have all lines aligned on notes.  This shift is applied to the right before every syllable and then to the left after placing an empty box.  At the beginning of the line the shift to the right is ignored by \TeX\ as leading white space, but the shift left is not because of the presence of the “character” of the empty box.
```

Comment:
What do you mean?  Are you questioning the length of the description or the fact that the whole paragraph appears on a single line?

If the former, how would you suggest shortening the description?

If the latter, of course it doesn't.  It could be hard wrapped to something else as TeX treats end-of-lines as spaces.  My editor simply defaults to soft wrapping its ability to "unwrap" is, well, non-functional (everything gets put on the same line).  As a result, I seldom take the time to perform the extra hard wrapping step when creating/editing files.